test/math_test.py

```diff
@@ -1577,6 +1577,13 @@ def test_pickle(self):
         self.assertEqual(pickle.loads(pickle.dumps(v2)), v2)
         self.assertEqual(pickle.loads(pickle.dumps(v3)), v3)
 
+    def testVector2Subclass(self):
```

Comment:
Can you please add a decorator to show that this is an expected failure?
Please see the documentation here on how to do that: https://docs.python.org/3/library/unittest.html#unittest.expectedFailure

This way the tests will pass on TravisCI and Appveyor.

Reply:
Yup, I'll do that as soon as I can. Thanks for the help